Changelog.md

```diff
@@ -16,6 +16,7 @@ Bugfixes:
  * SMTChecker: Fix internal error in sequential storage array pushes (``push().push()`).
  * Type Checker: Fix overload resolution in combination with ``{value: ...}``.
  * Type Checker: Fix internal compiler error related to oversized types.
+ * Type Checker: Fixing type deduction issues on function types.
```

Comment:
I think this should be more specific and mention function calls with named arguments

Reply:
I changed it, and was a bit more specific. Hope it looks better now :)